presto-clickhouse/src/main/java/com/facebook/presto/plugin/clickhouse/ClickHouseMetadata.java

```diff
@@ -272,4 +277,10 @@ public void dropSchema(ConnectorSession session, String schemaName)
     {
         clickHouseClient.dropSchema(ClickHouseIdentity.from(session), schemaName);
     }
+
+    @Override
+    public String normalizeIdentifier(ConnectorSession session, String identifier)
+    {
+        return clickHouseConfig.isCaseSensitiveNameMatching() ? identifier : identifier.toLowerCase(ENGLISH);
```

Comment:
Can clickhouse support unicode identifer? If yes then I think we coukd use `should use
identifier.toLowerCase(Locale.ROOT)` instead also add few tests accrodingly.